.github/workflows/create-release.yml

```diff
@@ -137,20 +143,20 @@ jobs:
         name: "${{ matrix.arch }}-${{ env.PATCHED_USNS_FILENAME }}"
         path: "${{ matrix.arch }}-${{ env.PATCHED_USNS_FILENAME }}"
 
-  stack_files_changed:
```

Comment:
Can you add a little bit of context why we dont need this step. Also is not exported in any of the steps

Reply:
This change is also based on https://github.com/paketo-buildpacks/github-config/commit/825e683c70f404c62b966a3a78ab9b01881050a8. I see that there is `Determine If Stack Files Changed` job, that sets `stack_files_changed=true`. This is later used in [L445](https://github.com/paketo-buildpacks/jammy-tiny-stack/pull/174/files#diff-8ac33fe295df086d3e55df1eb01194819d34cc0f5f54076dba96ceac0e40bd79R445) where `needs.run_if_stack_files_changed.result == 'success'` became `needs.create_stack.outputs.stack_files_changed == 'true'`.

Reply:
The `stack_files_changed: ${{ steps.compare.outputs.stack_files_changed }}` is removed and on the next step you export it `echo "stack_files_changed=true" >> "$GITHUB_OUTPUT"` Can you validate that is actually exported and it has a value?